v5.5.sed

```diff
@@ -0,0 +1,2 @@
+s/@@VERSION@@/5.5/
```

Comment:
I'd pass that bridge when we get there, but I do not think we will ever get there. IMO `@@XYZ@@` is pretty much a standard for substitutions simple as these. If we end up needing something different, it won't be hard to run sed on the whore repository and changing that. But again, I think this will be enough for us. At least I'm hoping python won't be needing something super-complicated with it's 4 versions :)